docs/src/content/docs/for-administrators/configuring-extra-files.md

```diff
@@ -2,12 +2,25 @@
 title: Configuring extra file submission
 ---
 
-Loculus supports the handling of arbitrary files.
-You can configure Loculus to support the submission of extra files for sequences, as well as providing extra files along with the sequence data and metadata for download. A typical usecase would be for raw reads. The files are stored in S3.
+Loculus supports the handling of arbitrary files associated with sequence entries.
+You can configure Loculus to support the submission of extra files for sequences, as well as providing extra files along with the sequence data and metadata for download. A typical usecase would be for raw reads. The files are stored in [S3](../../introduction/glossary#s3-simple-storage-service).
+
+To enable this feature you need to configure an S3 bucket for Loculus to use, and then configure the file categories per organism.
+
+## Conceptional overview
+
+Extra files submitted alongside sequence and metadata are treated differently. Loculus uses S3 - a generic object storage service - to store these files. Files are uploaded directly to S3 using presigned URLs.
+
+Unlike files that contain sequence data, the file sharing files are not inherently coupled to any particular sequence.
+Files are uploaded first, and then associated to a sequence entry, at the time of uploading, only an owning group needs to be specified. Because of this, the same file can also be attached to multiple sequence entries as well.
```

Comment:
```suggestion
Files are uploaded first, and then associated to a sequence entry; at the time of uploading, only an owning group needs to be specified. Because of this, the same file can also be attached to multiple sequence entries as well.
```